src/store/index.js

```diff
@@ -170,8 +170,13 @@ function handleRegistrationVerify({ dispatch }) {
         credentials: 'include',
       })
         .then((res) => {
-          if (res.status !== 204) throw res.status;
-          dispatch({ type: 'registrationVerified' });
+          if (res.status === 204) {
+            dispatch({ type: 'registrationVerified' });
+          } else if (res.status === 400) {
+            dispatch({ type: 'tokenInvalid' });
```

Comment:
Maybe it's nice to actually check for the `Unknown email verification token` message such that other 400 errors are still visible as general errors.
![image](https://user-images.githubusercontent.com/17035191/217535632-141e91c1-3058-4962-9e7e-cc3b77c1276f.png)

Reply:
@ivard Good one, but I'm not sure here: because if we would change the message on the myirma server one day (which could happen more likely compared to the status code I think), we would definitely have forgotten about this specific check. And all other 40x messages fall through and get thrown as a generic error. But you are true: it could happen the server returns a 400 status with another meaning.

Did some digging: What if we make the myirma server return 401 and make the check on that in case of an invalid token? It seems that's a commonly used status code, see: https://www.google.com/search?q=http+status+code+expired+token.

Of course this means a small change in irmago. 

What do you think of this?